peer/brontide.go

```diff
@@ -738,6 +746,12 @@ func (p *Brontide) Start() error {
 		return err
 	}
 
+	// Register the message router now as we may need to register some
+	// endpoints while loading the channels below.
+	p.msgRouter.WhenSome(func(router msgmux.Router) {
+		router.Start()
+	})
```

Comment:
I really wish that go would let you just do `p.msgRouter.WhenSome(Router.Start)` but alas...